integration_tests/e2e/order/monitoring-conditions/monitoringConditions.cy.ts

```diff
@@ -130,17 +164,22 @@ context('Monitoring conditions main section', () => {
       cy.get('select[name="orderType"]').select('immigration')
       cy.get('select[name="orderTypeDescription"]').select('GPS Acquisitive Crime HDC')
       cy.get('select[name="conditionType"]').select('License Condition of a Custodial Order')
-      cy.get('#startDate-startDay').type('27')
-      cy.get('#startDate-startMonth').type('3')
-      cy.get('#startDate-startYear').type('2024')
-      cy.get('#endDate-endDay').type('28')
-      cy.get('#endDate-endMonth').type('4')
-      cy.get('#endDate-endYear').type('2025')
+      cy.get('#startDate-day').type('27')
+      cy.get('#startDate-month').type('3')
+      cy.get('#startDate-year').type('2024')
+      cy.get('#startDate-hours').type('01')
+      cy.get('#startDate-minutes').type('02')
+      cy.get('#endDate-day').type('28')
+      cy.get('#endDate-month').type('4')
+      cy.get('#endDate-year').type('2025')
+      cy.get('#endDate-hours').type('03')
+      cy.get('#endDate-minutes').type('04')
       page.form.saveAndContinueButton.click()
       Page.verifyOnPage(InstallationAddressPage)
       cy.task('getStubbedRequest', `/orders/${mockOrderId}/monitoring-conditions`).then(requests => {
         expect(requests).to.have.lengthOf(1)
         expect(requests[0]).to.deep.equal({
+          action: 'continue',
```

Comment:
I don't think we need to send `action` to the api

Reply:
`action` is no longer sent to the api.